shell/init.go

```diff
@@ -11,8 +11,9 @@ ANTIBODY_BINARY="%s"
 antibody() {
 	case "$1" in
 	bundle|update)
+		tmp_dir=$(mktemp -d)
```

Comment:
this will make it nearly impossible to find where the logs are, since each execution will create a new dir...

I guess we can use the `TMPDIR` variable directly, and then update the docs to point that out... what do you think?